### ISSUE TITLE
build_visit mili + fortran issue

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1169]
on osx, mili's configure fails when it can't find gfortran

I tried setting FC_COMPILER to point to gfortran, mili's configure still failed.
It worked only after I added gfortran to my path. 

noting here b/c we may want to improve our build_visit support for mili when we move to use the open source mili release

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 06/20/2016 11:40 pm
Original update: 06/22/2016 02:23 pm
Ticket number: 2633
